examples/pytorch/llm/scripts/qwen_72b_chat/torchacc/lora_fsdp_sft.sh

```diff
@@ -0,0 +1,27 @@
+export USE_TORCHACC=1
+export XLA_FLAGS='--xla_gpu_enable_latency_hiding_scheduler=true --xla_enable_pt_all_reduce_optimizer=false --xla_gpu_enable_async_all_gather=true --xla_gpu_enable_async_reduce_scatter=true --xla_multiheap_size_constraint_per_heap=4294967296 --xla_disable_hlo_passes=rematerialization,all-gather-combiner,all-reduce-combiner,reduce-scatter-combiner'
```

Comment:
Maybe we should move the ENV variables from Line2 to Line6 to TorchAcc.

Reply:
Removed certain flags that are pre-configured within TorchAcc.